synapseclient/core/remote_file_storage_wrappers.py

```diff
@@ -72,7 +75,7 @@ def download_file(
         *,
         profile_name: str = None,
         credentials: typing.Dict[str, str] = None,
-        show_progress: bool = True,
+        progress_bar: tqdm = None,
```

Comment:
nit: shouldn't `progress_bar` type-hint be `Union[tqdm, None]`

Reply:
Updated